embedchain/memory/base.py

```diff
@@ -97,7 +109,15 @@ def get(self, app_id, session_id, num_rounds=10, display_format=False) -> list[C
             metadata = self._deserialize_json(metadata=metadata)
             # Return list of dict if display_format is True
             if display_format:
-                history.append({"human": question, "ai": answer, "metadata": metadata, "timestamp": timestamp})
+                history.append(
+                    {
+                        "session_id": session_id,
```

Comment:
Why do you need the session_id as part of the history? Are we using it anywhere?

Reply:
Was thinking of using it in UI to club results/search using `session_id`

Reply:
I see. Ok sounds good.